src/routes/controllers/auth.controller.js

```diff
@@ -108,24 +108,40 @@ export async function socialController(req, res) {
 
   try {
     const tokenPayload = {
-      id: user.id,
-      email: user.email,
+      user: {
+        id: user.id,
+        email: user.email,
+        username: user.username,
+      },
     };
     const token = await generateToken(tokenPayload);
+    console.log(isNewRecord);
     if (isNewRecord) {
-      return successResponse(
-        res,
-        201,
-        'You have successfully registered however you would need to check your mail to verify your account',
-        {
-          token,
-        },
-      );
+      // return successResponse(
```

Comment:
Remove commented out code

Reply:
you commented when I was still working on this. Kindly delete thanks